controllers/nodehealthcheck_controller.go

```diff
@@ -134,6 +131,23 @@ func (r *NodeHealthCheckReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	return result, nil
 }
 
+func (r *NodeHealthCheckReconciler) shouldTryRemediation(nhc remediationv1alpha1.NodeHealthCheck, unhealthyNodes []v1.Node, maxUnhealthy int) bool {
+	if len(unhealthyNodes) == 0 {
+		return false
+	}
+	if len(unhealthyNodes) <= maxUnhealthy {
+		if len(nhc.Spec.PauseRequests) > 0 {
+			// some actors want's to pause remediation.
```

Comment:
nit: s/want's/wants

Reply:
maybe `if len(unhealthyNodes) > maxUnhealthy {` and handle that first?